fenix/app/src/main/res/xml/search_settings_preferences.xml

```diff
@@ -19,23 +19,30 @@
             android:title="@string/preferences_manage_search_shortcuts"
             android:summary="@string/preferences_manage_search_shortcuts_summary"/>
     </PreferenceCategory>
-
```

Comment:
nit: Could we keep this new line to help maintain that visual separation of our `<PreferenceCategory>`? This will mean also add a new line for for your new additions as well.